src/model/index.ts

```diff
@@ -66,6 +72,16 @@ class Model implements WorkbookModel {
     if ((value?.object_type ?? "").startsWith("TX_STOCK_")) {
       this.TX_STOCK(value);
     }
+
+    if ((value?.object_type ?? "").startsWith("TX_PLAN_")) {
```

Comment:
**Non-blocking:** I would extend this to `TX_PLAN_SECURITY_` because that is the full "prefix" here. We should probably also go ahead and allow things that start with `TX_EQUITY_COMPENSATION_` because of https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/pull/397

Reply:
Ok. I will fix it. Do objects of type `TX_EQUITY_COMPENSATION_` have the same structure as the `TX_PLAN_SECURITY_` ones?

Reply:
Yes, the last part is the same. Just make sure we support both.